pages/product_list.html

```diff
@@ -0,0 +1,389 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/stylesheets/main.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <title>Products | Accel Sports</title>
+</head>
+<body>
+  <header class="head as-header flex-justify-sp-bw">
+    <a href="/index.html" class="logo flex-center"
+    >Accel Sports</a>
+    <div class="search-bar">
+      <input type="text" class="search-bar__input" placeholder="Search...">
+      <button class="search-bar__btn">
+        <i class="fas fa-search"></i>
+      </button>
+    </div>
+    <nav class="nav">
+      <ul class="nav_ul flex-justify-sp-bw">
+        <li class="nav_li"><a href="">Login</a></li>
+        <li class="nav_li"><a href=""><i class="fa-regular fa-heart"></i></a></li>
+        <li class="nav_li">
+          <a href=""><i class=" fas fa-regular fa-cart-shopping"></i></a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="as-prod__main d-flex">
```

Comment:
one suggestion- the class names are not understandable in first go, try to use better names.

Reply:
you can work on it in next projects.